packages/core/src/prompt-system/ContextDetector.test.ts

```
@@ -31,7 +31,7 @@ describe('ContextDetector', () => {
 
   describe('detectTaskContext', () => {
     it('should detect basic context with defaults', () => {
-      vi.stubEnv('SANDBOX', undefined);
+      delete process.env.SANDBOX;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)